scripts/trustbloc_local_setup.sh

```diff
@@ -36,4 +36,6 @@ cat > ${SANDBOX_HOME}/hosts <<EOF
 127.0.0.1 issuer-adapter.trustbloc.local
 127.0.0.1 rp-adapter.trustbloc.local
 127.0.0.1 rp-adapter-hydra.trustbloc.local
+127.0.0.1 edv.trustbloc.local
+127.0.0.1 shared.couchdb
```

Comment:
While not strictly necessary, this will allow you to access the visual database viewer in your browser by going to http://shared.couchdb:5984/_utils/